TeamCode/src/main/java/suitbots/Robot.java

```diff
@@ -0,0 +1,65 @@
+package suitbots;
+
+import com.qualcomm.hardware.bosch.BNO055IMU;
+import com.qualcomm.hardware.bosch.JustLoggingAccelerationIntegrator;
+import com.qualcomm.robotcore.hardware.ColorSensor;
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+
+import org.firstinspires.ftc.robotcore.external.navigation.AngleUnit;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesOrder;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesReference;
+import org.firstinspires.ftc.robotcore.external.navigation.Orientation;
+
+/**
+ * Created by Samantha on 9/2/2017.
+ */
+
+public class Robot {
+    //Yo necesito dormir.
+    private double lastG;
+    private BNO055IMU gyro;
+    private DcMotor lf, lr, rf, rr;
+    private ColorSensor lineDetector;
+    public Robot(HardwareMap h) {
+        gyro = h.get(BNO055IMU.class, "gyro");
+        initilizeGyro();
+        lineDetector = h.colorSensor.get("lineDetector");
+
+    }
+
+    public void resetGyro() {
+        lastG = getGyroRaw();
+    }
+    public double getGyroRaw() {
+        Orientation angles   = gyro.getAngularOrientation(AxesReference.INTRINSIC, AxesOrder.ZYX, AngleUnit.RADIANS);
+        return angles.firstAngle;
+    }
+    public double getGyro(){
+        return (getGyroRaw()-lastG)%2*Math.PI;
```

Comment:
We didn't really test the gyro resetting yesterday, so this may be written incorrectly. The possible problem is that both the `*` and `%` operators have the same [precedence](http://introcs.cs.princeton.edu/java/11precedence/), so the computer may do the math differently than you expect.

Worth testing to see if this works out to

`((getGyroRaw() - lastG) % 2) * Math.PI` 

or

`(getGyroRaw() - lastG) % (2 * Math.PI)`

You could always group the last two explicitly -- by writing it with the parens where you want them to be -- to be sure.

Reply:
Yep, this definitely need to be grouped explicitly. If you could change this to `* (2.0 * Math.PI)` I'll go ahead and merge.